### DISCUSSION TITLE
Help us to enhance the code.

### DISCUSSION THREAD

**Discussion by lcpt** [xcfem/xc#112] [Q&A]
I have known the existence of [this page](https://lgtm.com/projects/g/xcfem/xc) thanks to @luzpaz

This page shows many small bugs, some of which are easy to fix (we'll deal with the more difficult ones).

Could you help us to fix them?

**Comment by ebrahimraeyat** [comment]
It is fantastic. I think if you use VSCode, many of this error showing when you code. Beside that, this is also useful:

https://www.sonarlint.org/

I like to work on it.

**Reply by lcpt** [reply]
Thanks, @ebrahimraeyat. I'm happy using Emacs to coding, but I find those tools quite useful as «post-processing» utilities.

**Comment by luzpaz** [comment]
I've also sent some typos upstream https://github.com/OpenSees/OpenSees/pull/821 to see if the OpenSees team are keen to merge typo fixes. It would make sense for them as quite a few downstream projects that leverage OpenSees.

**Reply by luzpaz** [reply]
Feel free to +1 the upstream PR to show support for the decision. OpenSees code has lots of documentation typos

**Reply by lcpt** [reply]
OK
